tests/dag_processing/test_job_runner.py

```diff
@@ -1460,3 +1460,20 @@ def test_launch_process(self):
         processor_agent._process.join()
 
         assert os.path.isfile(log_file_loc)
+
+    @conf_vars({("logging", "dag_processor_manager_log_stdout"): "True"})
```

Comment:
can you add the test case for when it is disabled?

Reply:
Sure. Added